nas.py

```diff
@@ -69,6 +68,21 @@ def copy(self, remote_user, remote_ip, remote_path, local_path, to_local):
         stdout, stderr = p.communicate()
         return stdout, stderr, p.returncode
 
+    def copy_to_restricted(self, user, ip, from_path, to_path,
```

Comment:
Ok, got it. Idea: copy() is now more or less useless. Should we rename copy_to_restricted() to copy and make copy() private? copy() -> _copy()?

The new copy can later be extended by a new argument called **mode** with the options: **restricted** (default) and **non_restricted**.

It makes no sense to show every user copy internals! I mean the user has no gain to know that the function copy is somewhat special and the copy is handled over an indirection.